src/Apps/Auction/Routes/__tests__/ConfirmBid.test.tsx

```diff
@@ -268,6 +269,41 @@ describe("Routes/ConfirmBid", () => {
       )
     })
 
+    xit("tracks a success event to Segment including Criteo info", async done => {
+      const env = setupTestEnv()
+      const page = await env.buildPage()
+      env.mutations.useResultsOnce(createBidderPositionSuccessfulAndBidder)
+      mockBidderPositionQuery.mockResolvedValue(
+        confirmBidBidderPositionQueryWithWinning
+      )
+
+      await page.submitForm()
+
+      setTimeout(() => {
+        // TODO: What's up with it being called 2 times?
```

Comment:
This was caused by the previous test that had a mock with bidding bidding status: https://github.com/artsy/reaction/pull/2990/commits/c07a458efc5b21f978ae66c508f8f7c6de74bd19#diff-97e6a9881aac57a032e2b138e21443fcR251
Because the bidding status was pending the polling logic was getting kicked in, but the test finishes before the delayed logic is executed. I changed the mock data to the bidding status of `winning` so it'll immediately call the tracking code, which makes the test execution order more predictable.